x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/use_view_in_discover_action.tsx

```diff
@@ -28,8 +28,8 @@ export const useViewInDiscoverAction = (dataViewId?: string): TableItemAction =>
   const { application, share, data } = useAiopsAppContext();
 
   const discoverLocator = useMemo(
-    () => share.url.locators.get('DISCOVER_APP_LOCATOR'),
-    [share.url.locators]
+    () => share?.url.locators.get('DISCOVER_APP_LOCATOR'),
```

Comment:
The locators don't seem to be working here:

<img width="356" alt="Screenshot 2024-10-10 at 10 32 39" src="https://github.com/user-attachments/assets/1c8dab1a-f0bc-44a9-80d5-f730f82037c7">

Reply:
I was able to fix that one in [41b4337](https://github.com/elastic/kibana/pull/192167/commits/41b4337f9ac15103c6b151c94cfb6b7f944aeb89).